### DISCUSSION TITLE
Solution for better awareness of ready status

### DISCUSSION THREAD

**Discussion by drewhousman1010** [ParabolInc/parabol#5584] [Feedback]
We hear from lots of folks that they have a hard time knowing when their team is ready to advance to the next phase, and they have a hard time remembering to hit the ready button. 

The most recent case was a [user ](https://app.hubspot.com/contacts/3888472/contact/20491151) 🔒  saying:

> One thing I hear every time I a new person uses Parabol is that they either can't tell when their team is ready to move on or they don't realize they should be hitting the ready button

Related to issues
* https://github.com/ParabolInc/parabol/issues/5420
* https://github.com/ParabolInc/parabol/issues/4951

**Comment by Dschoordsch** [comment]
Also related:
* #6141

**Comment by drewhousman1010** [comment]
A [user](https://app.hubspot.com/contacts/3888472/contact/55593051) 🔒 writes: 

> I’ve only used parabol a handful of times so far but one thing that I think could definitely use an improvement was the UX around clicking ready in-between steps. Many times I have to remind team members to click ready when they are done and it’s often hard to tell if ready was clicked or not. Maybe just making the buttons a little bigger and have better contrasting states between ready being checked or not. Some prompting text could be nice as well, to help guide people as to what to do during a step. Pass out likes, add pros / cons, click ready, etc.

**Comment by acressall** [comment]
A user writes:

> The "Ready"-button on the bottom of the view didn't give me a clear visual feedback after pressing it.

**Comment by acressall** [comment]
A user writes:

> Make the Ready/Next button more attention getting.

**Comment by acressall** [comment]
A user requests:

> Make it more clear that I've selected the "Ready" green check after each activity (I couldn't read the indicator to tell that I had checked it or not)

**Comment by acressall** [comment]
A user writes:

> The ready button isn't clear that it should be clicked.

**Comment by lilicaruso** [comment]
A user says:

> They have to remind meeting participants to click the Ready button, it is not obvious that it needs to be clicked.
